src/confcom/azext_confcom/tests/latest/test_confcom_kata.py

```diff
@@ -58,13 +57,24 @@ def test_invalid_config_map_path(self):
         os.remove(filename)
         self.assertNotEqual(wrapped_exit.exception.code, 0)
 
-    def test_invalid_settings(self):
+    def test_valid_settings(self):
         filename = "pod2.yaml"
         with open(filename, "w") as f:
             f.write(KataPolicyGen.pod_string)
         with self.assertRaises(SystemExit) as wrapped_exit:
             katapolicygen_confcom(
-                filename, None, settings_file_name="genpolicy-settings.json"
+                filename, None
             )
+        with open(filename, "r") as f:
+            content = f.read()
         os.remove(filename)
```

Comment:
nit: `try-finally` can be introduced when a test has to do some clean up. `tearDownClass` could be an alternative